src/qa4sm_preprocessing/level2/smos.py

```diff
@@ -160,6 +160,8 @@ def _global_metadata(self, fname: Union[Path, str]):
         }
 
     def repurpose(self, *args, **kwargs):
+        if not kwargs.get('img2ts_kwargs', None):
+            kwargs['img2ts_kwargs'] = dict(backend='multiprocessing')
         return super().repurpose(*args, **kwargs)
```

Comment:
@wpreimes I added the solution you provided me with here. It looks like it fixes the issue, but locally it doesn't work for me :(

Reply:
Not much I can do without an error. Maybe try to re-install repurpose or multiprocessing. Anyway, I assume that local issues are not directly related by this package. So feel free to merge and release a new version if necessary